packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -38,34 +73,61 @@ export function DiscoverSection({
           </span>
         ),
       }),
-    },
-    {
-      icon: (active: boolean) => (
-        <ListIcon Icon={() => <HotIcon secondary={active} />} />
-      ),
-      title: 'Popular',
-      path: '/popular',
-      action: () => onNavTabClick?.('popular'),
-    },
-    {
-      icon: (active: boolean) => (
-        <ListIcon Icon={() => <UpvoteIcon secondary={active} />} />
-      ),
-      title: 'Most upvoted',
-      path: '/upvoted',
-      action: () => onNavTabClick?.('upvoted'),
-    },
-    {
-      icon: (active: boolean) => (
-        <ListIcon Icon={() => <SearchIcon secondary={active} />} />
-      ),
-      title: 'Search',
-      hideOnMobile: true,
-      path: '/search',
-      action: enableSearch,
-      showActiveAsH1: true,
-    },
-  ];
+    };
+
+    if (!isV1Sidebar) {
+      return [
+        discussion,
+        feeds,
+        {
+          icon: (active: boolean) => (
+            <ListIcon Icon={() => <UpvoteIcon secondary={active} />} />
+          ),
+          title: 'Most upvoted',
+          path: '/upvoted',
+          action: () => onNavTabClick?.('upvoted'),
+        },
+        {
+          icon: (active: boolean) => (
+            <ListIcon Icon={() => <SearchIcon secondary={active} />} />
+          ),
+          title: 'Search',
+          hideOnMobile: true,
+          path: '/search',
+          action: enableSearch,
+          showActiveAsH1: true,
+        },
+      ];
+    }
+
+    return [
+      feeds,
+      discussion,
+      {
+        icon: (active: boolean) => (
+          <ListIcon Icon={() => <HashtagIcon secondary={active} />} />
+        ),
+        title: 'Tags',
+        path: '/tags',
+        action: () => locationPush('/tags'),
+      },
+      {
+        icon: (active: boolean) => (
+          <ListIcon Icon={() => <EarthIcon secondary={active} />} />
+        ),
+        title: 'Sources',
+        path: '/sources',
+        action: () => locationPush('/sources'),
+      },
```

Comment:
Let's add leaderboard already :)

Reply:
that would be /leaderboard right @rebelchris ?

Reply:
It's `/users`